Api/Modpacks.cs

```diff
@@ -381,7 +383,8 @@ static class Modpacks {
             } else if ((string)btn.Tag == "btn2") {
                 OpenFileDialog ofd = new OpenFileDialog {
                     CheckFileExists = false,    // allow modpack creators to type in a filename for creating new files
-                    InitialDirectory = Config.MCC_home
+                    InitialDirectory = Config.MCC_home,
+                    Title = "Select a matching file to overwrite in your game:"
```

Comment:
This dialog can also be used to create a new file. It's not very obvious and I plan to add a help tip to explain, but users can type in a file name after navigating to a folder they want a new file to be created in, and type a name in the bar to have the modpack create a new file. Lets change this to "Select a destination"

Reply:
Gotcha. Sounds good for now, done